jquery.highlight-within-textarea.js

```diff
@@ -53,18 +53,50 @@
 			return 'other';
 		},
 
+		getHighlightsDivCssFix: function(currentTextarea) {
+			const textareaStyle = window.getComputedStyle(currentTextarea);
+			return {
+				'font-size': textareaStyle.getPropertyValue('font-size'),
+				'font-family': textareaStyle.getPropertyValue('font-family'),
+				'line-height': textareaStyle.getPropertyValue('line-height'),
+				'padding': textareaStyle.getPropertyValue('padding')
+			};
+		},
+
+		getBackdropDivCssFix: function(currentTextarea) {
+			const textareaStyle = window.getComputedStyle(currentTextarea);
+			return {
+				'background-color': textareaStyle.getPropertyValue('background-color')
+			};
+		},
+
+		getContainerDivCssFix: function(currentTextarea) {
+			const textareaStyle = window.getComputedStyle(currentTextarea);
```

Comment:
Is there a performance hit with `getComputedStyle`? Worth caching?

Reply:
Fixed.